hack/make/tests/e2e.mk

```diff
@@ -15,7 +15,8 @@ test/e2e/cloudnative: manifests/crd/helm
 	go test -v -tags e2e -timeout 30m -count=1 ./test/scenarios/cloudnative/basic
 
 ## Runs CloudNative e2e test only
```

Comment:
The comment indicates that CloudNative tests are run, so maybe you could update the comments as well

Reply:
good catch